tests/Test/AbstractFixerTestCase.php

```diff
@@ -379,28 +379,26 @@ final public function testProperMethodNaming(): void
         }
 
         $exceptionGroup = [
-            'CastNotation',
-            'ClassNotation',
             'ClassUsage',
             'Comment',
             'ConstantNotation',
             'ControlStructure',
             'DoctrineAnnotation',
-            'FunctionNotation',
+            'FunctionNotation', // remove me
             'Import',
-            'LanguageConstruct',
+            'LanguageConstruct', // remove me
             'ListNotation',
             'NamespaceNotation',
             'Naming',
-            'Operator',
+            'Operator', // remove me
             'PhpTag',
             'PhpUnit',
             'Phpdoc',
             'ReturnNotation',
             'Semicolon',
             'Strict',
             'StringNotation',
-            'Whitespace',
+            'Whitespace', // remove me
```

Comment:
this removal should be part of PR, or ... ?

Reply:
According to the title yes, but now I'm wondering it it is already big enough.

Reply:
If you plan it only as separated PR, then consider to add `@TODO` or simply even drop the comment.

(PS i already have 21/26 files reviewed, only noticed those "missing cases" and tucked a bit on it)